plus/tool/src/main/webapp/WEB-INF/templates/form.html

```diff
@@ -182,6 +182,21 @@ <h1><span th="#{plus.tool.tenant.title}">Tenant:<span>
     let inp = document.getElementById(id+'-boolean');
     inp.value = checked ? 'true' : 'false';
 }
+</script>
+
+<script th:inline="javascript">
+    function validateTenantForm() {
+        let issuerInput = document.getElementById("issuer");
+        let issuerValue = issuerInput.value.trim();
```

Comment:
You can do it in one line, `const issuerValue = document.getElementById("issuer").value.trim();`. One less variable.

Reply:
LGTM. I can remove that.